.test.endpoints.json

```diff
@@ -0,0 +1,57 @@
+{
+  "endpoints": [
+    {
+      "name": "broker1",
+      "url": "http://127.0.0.1:8161",
+      "auth": [
+        {
+          "scheme": "basic",
+          "data": {
+            "username": "guest",
+            "password": "guest"
+          }
+        }
+      ]
+    },
+    {
+      "name": "broker2",
+      "url": "http://127.0.0.2:8161",
+      "auth": [
+        {
+          "scheme": "basic",
+          "data": {
+            "username": "guest",
+            "password": "guest"
+          }
+        }
+      ]
+    },
+    {
+      "name": "broker3",
+      "url": "http://127.0.0.3:8161",
+      "auth": [
+        {
+          "scheme": "basic",
+          "data": {
+            "username": "guest",
+            "password": "guest"
```

Comment:
We usually tend to avoid storing a fully decrypted password in a database and instead we tend to store its SHA256 representation. Upon login the function that must validate the user's password takes it and hashes it to finally compare the hashes.
That way, if a DB were to be leaked somewhere important credentials wouldn't be necessarily compromised.

Reply:
Yes you are right about the clear text password. It should be stored with some sort of encryption. Considering the api server could use it to login jolokia it may require a symmetric encryption algorithm.